cwe_api/main.py

```
@@ -24,6 +25,12 @@ async def classify_vulnerability(vuln_request: VulnerabilityRequest):
     result = inferencer(vuln)
     return {"result": result}
 
+@app.post("/cvss")
+async def classify_vulnerability(vuln_request: VulnerabilityRequest):
```

Ruff

Missing return type annotation for public function `classify_vulnerability`

(ANN201)